client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -998,14 +1033,22 @@ const ConnectedPlanFeatures2023Grid = connect(
 					isWpcomEnterpriseGridPlan( plan ) && planConstantObj.getPathSlug
 						? planConstantObj.getPathSlug()
 						: planObject?.product_name_short ?? '';
-				const storageOptions =
+				const storageFeatures =
 					( planConstantObj.get2023PricingGridSignupStorageOptions &&
 						planConstantObj.get2023PricingGridSignupStorageOptions(
 							showLegacyStorageFeature,
 							isCurrentPlan
 						) ) ||
 					[];
+				const storageOptions = storageFeatures.map( ( feature ) => {
+					const featureObject = getFeatureByKey( feature );
 
+					return {
+						key: feature,
```

Comment:
This data structure looks great :-)

Maybe `key`` could be renamed to `somethingSlug`, like we have for plans - but maybe not if "feature slugs" are not a common concept.

Reply:
Feature slugs are a common concept 👍 https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plans/features-list.tsx#L328

Does just `slug` work? The data structure is named storageOptions, so when we access slugs, it will typically look like `storageOption.slug`.

Reply:
Switched to `slug` in [f10579f](https://github.com/Automattic/wp-calypso/pull/79041/commits/f10579f4550cb5cb359b5e56bd4a015de73aefc4)